diff --git a/hooks/post_gen_project.py b/hooks/post_gen_project.py
index 3437158..f6191a9 100644
--- a/hooks/post_gen_project.py
+++ b/hooks/post_gen_project.py
@@ -2,6 +2,7 @@ import os
 import platform
 import shutil
 import subprocess
+import sys
 
 shutil.rmtree('licenses')
 
@@ -9,6 +10,14 @@ shutil.rmtree('licenses')
 os.unlink('{{ cookiecutter.project_slug }}/fire_workarounds.py')
 {%- endif %}
 
+print('Setting up virtual environment', file=sys.stderr)
+subprocess.run(
+    'pyenv exec python -m venv .venv',
+    check=True,
+    shell=True,
+    stdout=sys.stderr,
+)
+
 {%- if cookiecutter.install_dependencies_now == "y" %}
 poetry_environment = os.environ.copy()
 poetry_environment.update({
