diff --git a/hooks/post_gen_project.py b/hooks/post_gen_project.py
index 5c38461..38878e9 100644
--- a/hooks/post_gen_project.py
+++ b/hooks/post_gen_project.py
@@ -10,7 +10,7 @@ shutil.rmtree('licenses')
 os.unlink('{{ cookiecutter.project_slug }}/fire_workarounds.py')
 {%- endif %}
 
-print('Setting up virtual environment', file=sys.stderr)
+{# print('Setting up virtual environment', file=sys.stderr)
 subprocess.run(
     'pyenv exec python -m venv .venv',
     check=True,
@@ -44,4 +44,4 @@ except subprocess.CalledProcessError as e:
         file=sys.stderr,
         sep='\n',
     )
-{%- endif %}
+{%- endif %} #}
